content/blog/2020-call-for-proposals-announcement.md

```diff
@@ -0,0 +1,74 @@
+Date: 2020-05-09 10:00
+Author: Vipul Gupta
+Category: 2020
+
+That time is upon us again, folks. PyCon India is back and the call for proposals ([https://in.pycon.org/cfp/2020/proposals/](https://in.pycon.org/cfp/2020/proposals/)) has officially opened.
+
+<!-- PELICAN_END_SUMMARY -->
+<div class="text-center">
+  <img src="https://media.giphy.com/media/kgsBIWtPd5Q5Pw11Rq/giphy.gif" height="500px" width="400px">
+</div>
+
+2020 has been a rollercoaster for all of us. As this year's PyCon India 2020 is being organized online ([Announcement tweet!](https://twitter.com/pyconindia/status/1253732245635952643?s=20)), we want your support to make the conference bigger, better, and more accessible to all. After all, PyCon India is made by you, for you and with you at its center.
+
+So, whether you are someone submitting a proposal for talks for the first time or just want to share what you have been up to with Python this past year. This is YOUR time to shine!
+
+> Divided by COVID. Connected by Python!
+
+We are looking forward to seeing beginner, intermediate, and advanced proposals on topics mentioned below from beginner, intermediate, and advanced speakers to present those talks. You don’t need to be a 10-year veteran who has spoken at dozens of conferences. For making our conference truly diverse, we are looking for all kinds of people. That’s what our community is comprised of, and that’s what this conference’s schedule should be made from.
+
+Please make note of some important dates as mentioned below:
+
+- CFP opens on **24th April 2020**
+- CFP closes on **14th August 2020**
+- Schedule shall be released on **1st September 2020**
+- Talks will be presented online on **2nd and 3rd October 2020**
+
+## Where should you submit your proposal?
+
+After signing up for an account on [https://in.pycon.org/cfp/2020/proposals/](https://in.pycon.org/cfp/2020/proposals/), you will be redirected to create a **New Proposal** page which is where you can create and submit your proposal.
+
+<div class="text-center">
+  <img src="https://media.giphy.com/media/o0vwzuFwCGAFO/giphy.gif" height="650px" width="200px">
+</div>
+
+*Do make sure to read the best practices before you start typing them up your proposals especially if you are a first time speaker!*
+
+## Guidelines for creating the video preview
+
+With PyCon India 2020 being online this year, we are trying to introduce preview videos that participants can upload while submitting their proposals. These **preview videos** are about you talking about your talk, topics you intend to cover, and how you intend to cover them that our reviewers will take into account before making final decisions on your talk proposal. Please keep in mind to strictly follow all guidelines for creating your video preview as mentioned below:-
+
+**Duration & orientation**
+1 -3 Minute video to be recorded. The video should be recorded preferably in portrait mode.
+
+**Upload to YouTube**
```

Comment:
I think it's fine to use any other video platform as well

Reply:
Keeping one standard platform is a good idea and would be helpful for the reviewers to review, I think.